sign/action.yml

```diff
@@ -0,0 +1,21 @@
+name: notation-sign
+description: notation sign with plugin
+inputs:
+  plugin_name:
+    description: name of the signing plugin
+    required: true
+  plugin_url:
+    description: download url of the signing plugin
+    required: true
```

Comment:
When downloading from a plugin URL, the corresponding checksum must be provided.